opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -1213,6 +1232,7 @@ namespace Opm
         // only for output
         std::vector<double> perfRatePolymer_;
         std::vector<double> perfRateBrine_;
+        std::vector<double> perfRateZFrac_;
 
```

Comment:
Why not use the perfRateSolvent_ ? That will cleanup the code IMO.

Reply:
Problem: Call to 'solventWellRate' is used without being protected by 'has_solvent' (e.g. 'StandardWell').

Reply:
I see. 
I suggest either adding if(!has_sovlent) return 0.0 in solventWellRate(...) or guards usage of solventWellRate with if(has_solvent)

Reply:
Ok.